raiden/event_handler.py

```diff
@@ -245,6 +245,15 @@ def handle_channelnew(self, state_change):
                 token_address,
                 channel_address,
             )
+            # This is little hacky, used when bootstraping a token network.
+            # See ConnectionManager for details.
+            have_bootstrap = ((participant1 == connection_manager.BOOTSTRAP_ADDR) or
+                              (participant2 == connection_manager.BOOTSTRAP_ADDR))
+            if not have_bootstrap:
+                self.raiden.start_health_check_for(
+                    participant2 if
+                    participant1 == self.raiden.address else
+                    participant1)
```

Comment:
> I thought it was neccessary to have at least one channel registered when creating a new token network,

Yes, this is required. The connection manager is the piece of code that will open connections to new channels, it will kick off on user demand and create a channel *on-chain* for starting the network.

The code that this PR is patching is for the python counter part, that is not really needed since BOOTSTRAP_ADDR is assumed to not exist.